src/sentry/static/sentry/app/views/integrationPipeline/awsLambdaCloudformation.tsx

```diff
@@ -136,94 +137,72 @@ export default class AwsLambdaCloudformation extends React.Component<Props, Stat
     return (
-      <div>
-        <StyledAlert type="info">
-          {t('It might take a minute for the CloudFormation stack to be created')}
-        </StyledAlert>
-        <IconGroup pluginId="aws_lambda" />
-        <InstallSentry>{t('Install Sentry on your AWS Account')}</InstallSentry>
-        {this.renderInstructions()}
-        <StyledTextField
-          name="accountNumber"
-          placeholder="599817902985"
-          value={accountNumber}
-          onChange={this.handleChangeArn}
-          onBlur={this.validateAccountNumber}
-          error={accountNumberError}
-          inline={false}
-          label={t('AWS Account Number')}
-        />
-        <StyledSelectField
-          name="region"
-          placeholder="us-east-2"
-          value={region}
-          onChange={this.hanldeChangeRegion}
-          options={this.regionOptions}
-          allowClear={false}
-          inline={false}
-          label={t('AWS Region')}
-        />
+      <React.Fragment>
+        <HeaderWithHelp docsUrl="https://docs.sentry.io/product/integrations/aws_lambda/" />
+        <StyledList symbol="colored-numeric" initialCounterValue={initialStepNumber}>
+          <ListItem>
+            <Heading>{t("Add Sentry's CloudFormation to your AWS")}</Heading>
+            <StyledButton priority="primary" external href={this.cloudformationUrl}>
+              {t('Configure AWS')}
+            </StyledButton>
+          </ListItem>
+          <ListItem>
+            <Heading>{t('Add AWS Account Information')}</Heading>
+            <StyledTextField
+              name="accountNumber"
+              placeholder="599817902985"
+              value={accountNumber}
+              onChange={this.handleChangeArn}
+              onBlur={this.validateAccountNumber}
+              error={accountNumberError}
+              inline={false}
+              label={t('AWS Account Number')}
+            />
+            <StyledSelectField
+              name="region"
+              placeholder="us-east-2"
+              value={region}
+              onChange={this.hanldeChangeRegion}
+              options={this.regionOptions}
+              allowClear={false}
+              inline={false}
+              label={t('AWS Region')}
+            />
+          </ListItem>
+        </StyledList>
         <FooterWithButtons
-          docsUrl="https://docs.sentry.io/product/integrations/aws_lambda/"
           buttonText={t('Next')}
           onClick={this.handleSubmit}
           disabled={submitting || !this.formValid}
         />
-      </div>
+      </React.Fragment>
     );
   };
 }
 
-const InstructionWrapper = styled('div')`
-  margin-left: 20px;
-`;
-
-const StyledStepHeading = styled(StepHeading)`
-  font-size: ${p => p.theme.fontSizeLarge};
-  margin: 10px 0 0 0;
+const StyledButton = styled(Button)`
+  margin: ${space(2)} 0 ${space(2)} 0;
 `;
 
 const StyledTextField = styled(TextField)`
-  padding: ${space(1)} 65px;
   border-bottom: none;
```

Comment:
can't you kill this with `stacked` and not need this?

Reply:
@EvanPurkhiser Need what exactly?

Reply:
`border-bottom: none`